### DISCUSSION TITLE
[macOS/Passthrough] macOS on iGPU?

### DISCUSSION THREAD

**Discussion by eversiege** [Coopydood/ultimate-macOS-KVM#57] [VFIO-PCI / Passthrough]
Hello. First things first - loving this project, love testing it, contributing to it wherever I can.

Recently wanted to get myself a macOS KVM (or raw Hackintosh) to be able to do personal projects and code for ULTMOS, yet right now I am not sure how to do it exactly. I've exhausted my resources to find on to successfully pass through an iGPU (i5 7500U laptop), including trying to Hackintosh manually via Dortania guide, to no avail.

Please, if anyone possesses any knowledge about passing through iGPU to macOS, please let me know here, and I will test it live.

**Comment by dblhlx** [comment]
This link (https://github.com/vivekmiyani/OSX_GVT-D) may help.

**Comment by dblhlx** [comment]
"Theoretically compatible with any 14nm chip(Skylake, Kaby lake, Coffee Lake, Amber Lake, Whiskey Lake, Comet Lake)."
https://github.com/patmagauran/i915ovmfPkg
